### PR TITLE
FaceID text are added

### PR DESCRIPTION
**Pull request by muhammedyalcin**

### CODE REVIEW

File: lib/fingerprint.js [hunk 1]
```diff
@@ -29,6 +30,7 @@ var tbPassword;
 var encryptionFunction = sameReturner;
 var decryptionFunction = sameReturner;
 var fingerprintCallback;
+var isIphoneX = (Screen.height === 812 ? true : false); //detect the iphoneX acc to screen height
```
Comment (alperozisik) [inline lib/fingerprint.js]: You need to check it is iOS too.
It can be any android device with same height

File: lib/fingerprint.js [hunk 2]
```diff
@@ -244,8 +246,8 @@ function loginWithFingerprint() {
     }
     else if (firstLogin && System.fingerPrintAvailable) { //first login
         alert({
-            title: global.lang.useFingerprintTitle || "Use Fingerprint",
-            message: global.lang.useFingerprintMessage || "Would you like to use fingerprint login on your future logins?",
+            title: (isIphoneX  ? "Use FaceID" : global.lang.useFingerprintTitle || "Use Fingerprint"),
```
Comment (alperozisik) [inline lib/fingerprint.js]: Regardless what it is written here as message, language files have the highest priority.
If the entry in language file is not present then the default text is displayed in English.

The loging here is wrong for that. I will write my recommendation; please update the rest accordingly.

```javascript
//define this on top part
const LOGIN_METHOD = isIphoneX? "FaceID": "Fingerprint";

//...

title: global.lang.useFingerprintTitle || `Use ${LOGIN_METHOD}`,
```

File: lib/fingerprint.js [hunk 3]
```diff
@@ -244,8 +246,8 @@ function loginWithFingerprint() {
...
+            message: (isIphoneX ? "Would you like to use faceID login on your future logins?" : global.lang.useFingerprintMessage || "Would you like to use fingerprint login on your future logins?"),
```
Comment (alperozisik) [inline lib/fingerprint.js]: here too

File: lib/fingerprint.js [hunk 4]
```diff
@@ -263,7 +265,7 @@ function loginWithFingerprint() {
                         useFingerprintLogin = false);
                     Data.setBooleanVariable(dataKeys.firstLogin,
                         firstLogin = true);
-                    return callback("User does not want to use fingerprint");
+                    return callback((isIphoneX ? "User does not want to use faceID" : "User does not want to use fingerprint"));
```
Comment (alperozisik) [inline lib/fingerprint.js]: here too

File: lib/fingerprint.js [hunk 5]
```diff
@@ -272,7 +274,7 @@ function loginWithFingerprint() {
         if (!System.fingerPrintAvailable)
             return callback("Fingerprint is not available");
         else
-            return callback("User does not choose to use fingerprint login");
+            return callback((isIphoneX ? "User does not choose to use faceID login"  : "User does not choose to use fingerprint login"));
```
Comment (alperozisik) [inline lib/fingerprint.js]: here too

File: lib/fingerprint.js [hunk 6]
```diff
@@ -290,17 +292,17 @@ function loginWithFingerprint() {
     function validateFingerPrint() {
         System.validateFingerPrint({
             android: {
-                title: global.lang.scanFingerprintTitle || "Login with fingerprint"
+                title: (isIphoneX ? "Login with faceID" : global.lang.scanFingerprintTitle || "Login with fingerprint")
```
Comment (alperozisik) [inline lib/fingerprint.js]: here too

File: lib/fingerprint.js [hunk 7]
```diff
@@ -290,17 +292,17 @@ function loginWithFingerprint() {
...
             },
-            message: global.lang.scanFingerprintMessage || "Scan your finger",
+            message: (isIphoneX ? "Scan your faceID" : global.lang.scanFingerprintMessage || "Scan your finger"),
```
Comment (alperozisik) [inline lib/fingerprint.js]: here too

File: lib/fingerprint.js [hunk 8]
```diff
@@ -290,17 +292,17 @@ function loginWithFingerprint() {
...
             onSuccess: function() {
                 return callback(null, {
                     password: tbPassword.text || storedPassword,
                     success: onSuccess
                 });
             },
             onError: function() {
-                return callback("Fingerprint validation failed");
+                return callback((isIphoneX ? "FaceID validation failed"  : "Fingerprint validation failed"));
```
Comment (alperozisik) [inline lib/fingerprint.js]: here too

**Comment by muhammedyalcin** [comment]
Changes are reviewed according to feedbacks. Could you please check new commits ? 

Thanks

**Pull request merged**